### PR TITLE
give the quetionnaire module a basic designed skin

### PR DESCRIPTION
**Pull request by chiangkeith**

### CODE REVIEW

File: api/actions/loadArticles.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ export function loadQuestionnaire(req, params = []) {
     const { API_PROTOCOL, QUESTIONNAIRE_HOST } = config
     let url = `${API_PROTOCOL}://${QUESTIONNAIRE_HOST}/questionnaire`
     let slug = typeof params[0] === 'string' ? params[0] : null
-    url = slug ? `${url}/${slug}/${slug}.json` : url
+    url = slug ? `${url}/${slug}/${slug}.json?v16` : url
```
Comment (hcchien) [inline api/actions/loadArticles.js]: v16?

File: src/components/questionnaire/Option.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ class Option extends Component {
           let { animateInterval, currOpacity, style } = this.state
           let tmp = currOpacity
```
Comment (hcchien) [inline src/components/questionnaire/Option.js]: tmp seems not a good variable name for the context.

File: src/containers/Questionnaire.js [hunk 3]
```diff
@@ -173,50 +238,81 @@ class Questionnaire extends Component {
...
+                                <Button value="看答案" />
+                              </div>
                             </div>
                           </div>
                         </div>
 
                         { _.map(questions, (quest, idx) => {
-                          const designatedAnsId = _.get(quest, [ 'designated_option' ])
-                          const ans = _.get(this.props, [ 'ans', idx ], null)
+                          const tmpDesignatedAnsId = _.get(quest, [ 'designated_option' ])
```
Comment (hcchien) [inline src/containers/Questionnaire.js]: tmp means?

File: api/actions/loadArticles.js (hunk 1 above)
Comment (chiangkeith) [reply api/actions/loadArticles.js]: it's just for cache. but i'd take it off.

File: src/components/questionnaire/Option.js (hunk 2 above)
Comment (chiangkeith) [reply src/components/questionnaire/Option.js]: yep, just revised it. thanks for advice.

File: src/containers/Questionnaire.js (hunk 3 above)
Comment (chiangkeith) [reply src/containers/Questionnaire.js]: just gave it better var name.

**Pull request merged**